R/pkg/R/functions.R

```diff
@@ -2876,7 +2876,8 @@ setMethod("randn", signature(seed = "numeric"),
 
 #' regexp_extract
 #'
-#' Extract a specific(idx) group identified by a java regex, from the specified string column.
+#' Extract a specific \code{idx} group identified by a Java regex, from the specified string column.
```

Comment:
Minor comment: It looks like `idx` was removed in the comment change in https://github.com/apache/spark/pull/14525 -- I'm not sure if it matters, but we could either keep it consistent with the Scala/Python doc or keep the `idx` if you think it helps.